Source/LogWriter.swift

```diff
@@ -49,15 +71,18 @@ open class ConsoleWriter: LogMessageWriter {
         case print, nslog
     }
 
+    public let modifiers: [MessageModifier]
```

Comment:
Docstring should be added here or on the protocol. Probably best on the protocol property b/c then all the conformers get the docstring for free.